packages/c/curl/package.yml

```diff
@@ -1,25 +1,40 @@
 name       : curl
 version    : 8.6.0
-release    : 85
+release    : 86
 source     :
-    - https://github.com/curl/curl/releases/download/curl-8_6_0/curl-8.6.0.tar.gz : 9c6db808160015f30f3c656c0dec125feb9dc00753596bf858a272b5dd8dc398
-homepage   : https://curl.haxx.se
+    - https://github.com/curl/curl/releases/download/curl-8_6_0/curl-8.6.0.tar.gz#curl.tar.gz : 9c6db808160015f30f3c656c0dec125feb9dc00753596bf858a272b5dd8dc398
+extract    : no
+homepage   : https://curl.se
 license    : MIT
-component  : system.base
+component  :
+    - system.base
+    - gnutls : binary.compat
+    - gnutls-32bit : binary.compat
+summary    :
+    - Get a file from an HTTP, HTTPS or FTP server
+    - gnutls : GnuTLS version of curl
+    - gnutls-32bit : GnuTLS version of curl
+description:
+    - curl is a client to get files from servers using any of the supported protocols. The command is designed to work without user interaction or any kind of interactivity. curl offers a busload of useful tricks like proxy support, user authentication, ftp upload, HTTP post, file transfer resume and more.
+    - gnutls : This package provides binary compatibility only and enables the running of content expecting a GnuTLS version of curl. It is forbidden to build against it.
+    - gnutls-32bit : This package provides binary compatibility only and enables the running of content expecting a GnuTLS version of curl. It is forbidden to build against it.
 emul32     : yes
 summary    : Get a file from an HTTP, HTTPS or FTP server
 description: |
     curl is a client to get files from servers using any of the supported protocols. The command is designed to work without user interaction or any kind of interactivity.  curl offers a busload of useful tricks like proxy support, user authentication, ftp upload, HTTP post, file transfer resume and more.
```

Comment:
Should this have been deleted?

Reply:
You saw nothing! [POCKET SAND](https://www.youtube.com/watch?v=QLpUq__iQqw)